script/release.sh

```diff
@@ -27,7 +27,7 @@ function build_project() {
 	builddir="$(dirname "$1")"
 
 	# Build with all tags enabled.
-	make -C "$root" COMMIT_NO= BUILDTAGS="seccomp selinux apparmor" static
+	make -C "$root" COMMIT_NO= BUILDTAGS="seccomp selinux" static
```

Comment:
We can just drop `BUILDTAGS` entirely from here, because since #2254 they are already set by the Makefile.

Reply:
fixed